src/cli/nondaemon_commands.rs

```diff
@@ -21,6 +22,41 @@ pub fn clean_up_after_daemon() -> Result<()> {
     Ok(())
 }
 
+pub async fn wait(api_client: &DaemonClient) -> Result<()> {
+    for n in 0..5 {
+        match api_client
+            .client
+            .get(api_client.get_url("/info"))
+            .send()
+            .await
+        {
+            // if timeout, retry
+            Err(e) => {
+                if e.is_timeout() || e.is_connect() {
```

Comment:
I believe negative matching remove the else arm?

`if !(e.is_timeout() || e.is_connect()) {
    bail!(e)
}`